service/policy/objects.proto

```diff
@@ -460,6 +460,48 @@ message RegisteredResourceValue {
   common.Metadata metadata = 100;
 }
 
+message Obligation {
```

Comment:
I think the Obligation requires in the proto its `repeated ObligationValue values` so you can call `ListObligations` or `GetObligation` and see its children.